Changelog for 47.17.1 (#1923)
forcedotcom/salesforcedx-vscode: packages/salesforcedx-vscode/CHANGELOG.md

# 47.17.1 - January 30, 2020

## Fixed

#### salesforcedx-vscode-apex

> - Update Apex Language Server error message to include we support Java 8 and Java 11 ([PR #1900](https://github.com/forcedotcom/salesforcedx-vscode/pull/1900))

Reviewer: ```suggestion
- Update Apex Language Server error message to include support for Java 8 and Java 11 ([PR #1900](https://github.com/forcedotcom/salesforcedx-vscode/pull/1900))
```

## Added

#### salesforcedx-vscode-apex

> - Set cursor on the first argument when inserting method from code completion ([PR #1913](https://github.com/forcedotcom/salesforcedx-vscode/pull/1913))

Author: I'm adding images and gifs to highlight some of the changes done. You'll have to "manually" update the image url if you want to preview them since I have the link using `develop` branch.

> - Set cursor on the first argument when inserting method from code completion ([PR #1913](https://github.com/forcedotcom/salesforcedx-vscode/pull/1913))

Reviewer 2: ```suggestion
- Tab between arguments when inserting method from code completion ([PR #1913](https://github.com/forcedotcom/salesforcedx-vscode/pull/1913))
```

  ![GIF showing Apex code completion with new tab fix](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode/images/47.17.1/autocomplete_tabs.gif)

- Disable Global Objects from autocompletion when invoked from a member type ([PR #1913](https://github.com/forcedotcom/salesforcedx-vscode/pull/1913))
  ![GIF showing autocompletion results without global objects](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode/images/47.17.1/autocomplete_no_global_objects.gif)

> - Show documentation for autocomplete items ([PR #1913](https://github.com/forcedotcom/salesforcedx-vscode/pull/1913))

Reviewer 2: ```suggestion
- Show documentation for autocomplete items when definitions have block comments ([PR #1913](https://github.com/forcedotcom/salesforcedx-vscode/pull/1913))
```

Want to make clear it's currently just for things the user has added comments for at the moment

  ![GIF showing autocompletion results with documentation](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode/images/47.17.1/autocomplete-docs.gif)

> - Update documentation format for hover ([PR #1913](https://github.com/forcedotcom/salesforcedx-vscode/pull/1913))

Reviewer 2: ```suggestion
- Update how documentation is rendered when hovering over a symbol ([PR #1913](https://github.com/forcedotcom/salesforcedx-vscode/pull/1913))
```
